__tests__/praxis/arthur-deviation-weights/unit.ts

```diff
@@ -55,34 +58,34 @@ describe('ArthurDeviationWeights Class: Unit', () => {
       it('gets dimensions from inputLayer', () => {
         const width = 2;
         const height = 3;
-        const mockLayer = {
+        const mockLayer: any = {
           width,
           height,
         };
-        const p = new ArthurDeviationWeights(mockLayer);
+        const p: any = new ArthurDeviationWeights(mockLayer);
```

Comment:
Same

Reply:
If I set the type ArthurDeviationWeights here then in next line the error for length will be there.. Is there any solution for that??

Reply:
This does have a type `ArthurDeviationWeights`